contrib/neovide/template.py

```diff
@@ -0,0 +1,92 @@
+pkgname = "neovide"
+pkgver = "0.12.1"
+pkgrel = 0
+build_wrksrc = "neovide"
+build_style = "cargo"
+hostmakedepends = [
+    "cargo",
+    "ninja",
+    "libcxx-devel-static",
+    "libcxxabi-devel-static",
+]
+makedepends = [
+    "fontconfig-devel",
+    "freetype-devel",
+    "libexpat-devel",
+    "libjpeg-turbo-devel",
+    "libpng-devel",
+    "zlib-devel",
+]
+checkdepends = ["neovim"]
+pkgdesc = "No Nonsense Neovim Client in Rust"
```

Comment:
this violates pkgdesc conventions in at least 3 ways (and nothing about this seems no-nonsense)

Reply:
oh wait this was written by past me

will fix

---

**TL;DR** yesterday i thought of writing a template for this package, and today I discovered that a template already exists! So I got to work trying to fix it's build without even looking at the rest of the stuff.

That's why I was initially building 0.11.2 instead of the latest 0.12.1 (if you saw that on IRC)

Reply:
"Graphical client for neovim" sounds ok?